.github/CODEOWNERS

```diff
@@ -1,5 +1,16 @@
-# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+# CODEOWNERS reference: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
-.github/workflows/*       @inferno-chromium @naveensrinivasan @azeemshaikh38 @laurentsimon
-*.md    @olivekl
-docs/checks/internal/checks.yaml    @olivekl
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# the following users/teams will be requested for
+# review when someone opens a pull request.
+# TODO(owners): For ease of management, this should eventually shift to a
+#               defined GitHub team instead of individual usernames
+* @azeemshaikh38 @inferno-chromium @justaugustus @laurentsimon @naveensrinivasan
+
+# Docs
+# TODO(owners): For ease of management, this should eventually shift to a
+#               defined GitHub team instead of individual usernames
+*.md @olivekl
+# TODO(owners): Should we simplify this to `docs/`?
+docs/checks/internal/checks.yaml @olivekl
```

Comment:
Yes just docs/ makes more sense here.

Reply:
> Yes just docs/ makes more sense here.

Fixed in https://github.com/ossf/scorecard/pull/1530/commits/adf5dc6c834a5bf1ac3ddae1b8329f429703bf26.